apis/src/pages/tournaments.rs

```diff
@@ -17,17 +17,34 @@ pub fn Tournaments() -> impl IntoView {
             api.tournament(TournamentAction::GetAll(Abstract));
         };
     });
+    let search = create_rw_signal(String::new());
```

Comment:
Just a fyi, create_..signals are getting deprecated in leptos 0.7, better to get used to using RwSignal::new and other Type::new .